contrib/JLightning/README.md

```diff
@@ -0,0 +1,14 @@
+Draft version of the JLightning rpc interface for c-lightning.
+
+Using this client library is is as simple as the following code:
```

Comment:
Even if someone is familiar with java, would be nice to have the actual command to build and launch: e.g.
```
$ javac <details-to-build>
$ java <details-to-run>
```

Reply:
On that note, have you though of using Gradle or Maven to handle building and dependency management? Preferably [Gradle](https://docs.gradle.org/current/userguide/getting_started.html)